app/web/package.json

```diff
@@ -115,6 +116,7 @@
     "@types/react": "^17.0.22",
     "@types/react-dom": "^17.0.9",
     "@types/react-gtm-module": "^2.0.1",
+    "@types/react-helmet": "^6.1.2",
```

Comment:
We can uninstall this package as `react-helmet-async` comes with its own types, and also this wouldn't match up with it anyway since the package name is different.

Reply:
removed